agendas/2024/09-Sep/19-week-3.md

```diff
@@ -113,6 +114,7 @@ cadence of progress we also have weekly meetings which tend to be more informal.
 1. Introduction of attendees (5m, Host)
 1. Determine volunteers for note taking (1m, Host)
 1. Review agenda (2m, Host)
+    1. Deprecate key usage (https://github.com/graphql/composite-schemas-spec/issues/51)     
```

Comment:
Should this be an agenda topic at the end of the file instead?

Reply:
Updated